akka-javasdk/src/main/java/akka/javasdk/workflow/Workflow.java

```diff
@@ -146,6 +163,14 @@ public void _internalSetup(S state) {
   }
 
 
+  public Optional<BackgroundProcess<S>> backgroundProcess(){
+    return Optional.empty();
+  }
+
+  protected final Optional<BackgroundProcess<S>> currentBackgroundProcess() {
```

Comment:
not sure if this should be optional, because maybe it is better to throw an exception when using it is in the wrong place, e.g. andThen section in step definition.